src/runner/pch.h

```diff
@@ -24,3 +24,4 @@
 #include <string>
 #include <common/common.h>
 #include <ProjectTelemetry.h>
+#include <cpprest/json.h>
```

Comment:
This can be removed now that we don't use anymore the json argument, right?

Reply:
Yes, I'll remove it.